bluemix/configuration/core_config/cf_config.go

```diff
@@ -153,6 +157,22 @@ func (c *cfConfig) writeRaw(cb func()) {
 	}
 }
 
+func (c *cfConfig) refreshToken(refreshToken string) (uaa.Token, error) {
```

Comment:
Callout: I added a seperate refreshToken in this package. I was having difficulty trying to reuse the RefreshToken method in plugin_context since it not really used outside of plugins. I am open to suggestions on a better approach

Reply:
I'm wondering if we still have to worry about CF tokens.. @boyang9527 - any input here ?

Reply:
we need since we still support CF